ex_feels/assets/js/timefeels.js

```diff
@@ -398,26 +110,28 @@ function makeChart (data) {
   // clipping to start chart hidden and slide it in later
   var rectClip = svg.append('clipPath')
     .attr('id', 'rect-clip')
+    .attr('fill', 'url(#upperGradient)')
     .append('rect')
       .attr('width', 0)
       .attr('height', chartHeight);
-
-  addAxes(svg, xAxis, yAxis, chartHeight);
+  
+  svg.append("text") // add chart title
+      .attr("x", (chartWidth / 2))             
+      .attr("y", 0 - (margin.top / 20))
+      .attr("text-anchor", "middle")  
+      .style("font-size", "20px")
+      .style('font-weight', 'bold')
+      .text(name);
+
+  addAxes(svg, xAxis, yAxis, chartHeight, name);
   drawPaths(svg, data, x, y);
   startTransitions(chartWidth, rectClip);
 }
 
-var parseDate  = d3.time.format('%Y-%m-%d').parse;
-
-var data = rawDataJSON.map(function (d) {
-  return {
-    date:  parseDate(d.date),
-    pct05: d.pct05 / 1000,
-    pct25: d.pct25 / 1000,
-    pct50: d.pct50 / 1000,
-    pct75: d.pct75 / 1000,
-    pct95: d.pct95 / 1000
-  };
+// Hit the bitfeels api for stats, log and make chart
+const url = "/bitfeels/api/stats"
+var response = d3.json(url, function (classifiers) {
```

Comment:
Why assign this to a `var`?

Reply:
idk, i guess it should be `const`?

Reply:
It doesn't need to be assigned to anything.

`d3.json(url, function (classifiers) {`

Reply:
hodor :-1: